src/main/resources/db/migration/common/V5.0__schema_user_counterparty.sql

```diff
@@ -0,0 +1,7 @@
+    alter table user_detail add counterparty_uuid raw(255) null;
+    alter table counterparty drop column user_id;
```

Comment:
Think about what will happen to the existent records in the prod after running this script.